src/rpdk/core/project.py

```diff
@@ -959,6 +959,22 @@ def __set_property_type(prop_type, single_type=True):
                     lambda markdown_value: f"List of {markdown_value}"
                 )  # noqa: E731
 
+                if "items" not in prop.keys():
+                    LOG.warning(
+                        'Warning: found schema property of array type with no "items" key; \n'
```

Comment:
Is this the best default? 

```json
"items": {
  "type": "string"
}
```

Reply:
It is not the default value here, it is just an example in the event the (hook's) author chooses to have an input property of `array` type, whereas its items are of the `string` desired type.